site/_config.yml

```diff
@@ -16,7 +16,7 @@
 title: Apache Bahir
 email: dev@bahir.apache.org
 description: > # this means to ignore newlines until "baseurl:"
-  Apache Bahir provides extensions to distributed analytic platforms such as Apache Spark.
+  Apache Bahir provides extensions to multiple distributed analytic platforms.
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "http://bahir.apache.org" # the base hostname & protocol for your site
 #twitter_username: ApacheBahir
```

Comment:
on line 32, this: `exclude: [README.md,Gemfile*,docs/spark/templates]`
should now be: `exclude: [README.md,Gemfile*,docs/*/templates]` to also exclude flink templates from generation